test/promql/tests/data_plane/gitops_sync_test.yaml

```diff
@@ -0,0 +1,86 @@
+evaluation_interval: 1m
+
+rule_files:
+  - prometheus.gitops_sync_alerts.yaml
+
+tests:
+  # Alerted cases:
+  - interval: 1m
+    input_series:
+
+      # GitOps service is experiencing application sync success rate of less than 95%, so alert
+      - series: 'argocd_app_info{sync_status="Synced"}'
+        values: '1 2 3 4 5 6'
```

Comment:
This metric cannot return anything but `1`. The possible values to have in test cases for this is either `1` or `_` (meaning, it was not scraped).

Reply:
I updated the tests to just use 1 and _